projects/WTI-UI/src/app/modules/core/abstract-services/i-websocket.service.ts

```diff
@@ -15,6 +15,7 @@ export abstract class IWebsocketService {
   abstract stopWebsocket(): void;
 
   incomingMessage(message: WebsocketMessage) {
+	console.log('Incoming Message of ${message.type}',message.type)
```

Comment:
I thought one of your comments indicated you were going to remove console logging?

Reply:
ok, I'll accept that.